crates/rpc/rpc-eth-api/src/helpers/error.rs

```diff
@@ -0,0 +1,88 @@
+//! Helper traits to wrap generic l1 errors, in network specific error type configured in
+//! [`EthApiTypes`](crate::EthApiTypes).
+
+use reth_rpc_eth_types::EthApiError;
+use revm_primitives::EVMError;
+
+/// Helper trait to wrap core [`EthApiError`].
+pub trait FromEthApiError: From<EthApiError> {
+    /// Converts from error via [`EthApiError`].
+    fn from_err<E>(err: E) -> Self
```

Comment:
let's be more explicit about the error conversion
```suggestion
    fn from_eth_err<E>(err: E) -> Self
```